### ISSUE TITLE
YouTube Music scrobbling: songs not being recognized

### ISSUE THREAD

**Issue by a-github-account** [web-scrobbler/web-scrobbler#3391]
**Describe the bug**
The scrobbler is unable to scrobble (red chrome extension icon)
**How to reproduce**
1. Put your computer to sleep while listening to a song in YouTube Music
2. Reopen your computer
3. Listen to any song on YouTube Music
4. Restart Chrome
5. Listen to any song on YouTube Music again

**Expected behavior**
For there to be a green checkmark and scrobble successfully
**Screenshots**

**Debug logs**
```
storage.LastFM = {
  "sessionID": "********************************",
  "sessionName": "**********"
}
wrapper.js:74 storage.LibreFM = {}
wrapper.js:74 storage.ListenBrainz = {}
wrapper.js:74 storage.Maloja = {}
wrapper.js:74 storage.LocalCache = {
  "1iKLf5fyr3M": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "Lust"
  },
  "2VYzXS_p1Bs": {
    "album": null,
    "albumArtist": null,
    "artist": "Alpha Wolf",
    "track": "Restricted (R18+)"
  },
  "6KOql9vlYSo": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "Failed"
  },
  "AmmJG9fBmB0": {
    "album": "forgetmenot",
    "albumArtist": null,
    "artist": "Five Pebbles",
    "track": "cat's tongue"
  },
  "BA0whkcpogg": {
    "album": null,
    "albumArtist": null,
    "artist": "Slipknot",
    "track": "Carve (1997)"
  },
  "DeJLzMXKLOk": {
    "album": "Hybrid Theory",
    "albumArtist": null,
    "artist": "Linkin Park",
    "track": "With You"
  },
  "ErXVvF5S7gs": {
    "album": null,
    "albumArtist": null,
    "artist": "Slipknot",
    "track": "(SIC)"
  },
  "Irg5GY0l_-Y": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "Step Into The Dark"
  },
  "JOFtl7NyroI": {
    "album": null,
    "albumArtist": null,
    "artist": "PLAGUE THROAT",
    "track": "Fallible Transgression"
  },
  "O7YAHQQQkcQ": {
    "album": null,
    "albumArtist": null,
    "artist": "Treyarch",
    "track": "Pareidolia"
  },
  "R7b1Iws_gTA": {
    "album": null,
    "albumArtist": null,
    "artist": "Slipknot",
    "track": "AOV"
  },
  "Vd3zFk8BUBs": {
    "album": null,
    "albumArtist": null,
    "artist": "Blast Damned Flame",
    "track": "Majestic"
  },
  "VkOMVW491wE": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "Polymer"
  },
  "WjnlwoKjnaY": {
    "album": null,
    "albumArtist": null,
    "artist": "On A Pale Horse",
    "track": "Walks With Death"
  },
  "aATStGrHjLg": {
    "album": null,
    "albumArtist": null,
    "artist": "Slipknot",
    "track": "Eyeless"
  },
  "b1q6gbbf8yw": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "Last Rites"
  },
  "fvfM7oq1uZk": {
    "album": null,
    "albumArtist": null,
    "artist": "Vice Grip Throttle",
    "track": "Gut Bucket"
  },
  "hMa4hZQbrms": {
    "album": null,
    "albumArtist": null,
    "artist": "Toby Fox",
    "track": "ASGORE"
  },
  "jX8RTjUUcLE": {
    "album": null,
    "albumArtist": null,
    "artist": "Crystal Lake",
    "track": "Apollo"
  },
  "sNz4P9-aF5A": {
    "album": null,
    "albumArtist": null,
    "artist": "Slipknot",
    "track": "Confessions"
  },
  "wmEU-VypsHo": {
    "album": null,
    "albumArtist": null,
    "artist": "Treyarch",
    "track": "Dead Ended"
  },
  "yMUNq7I6A2E": {
    "album": null,
    "albumArtist": null,
    "artist": "United Mutation",
    "track": "Combat Boots"
  },
  "yp2qa66ifDI": {
    "album": null,
    "albumArtist": null,
    "artist": "t+pazolite",
    "track": "Boogie Woogie Splatter Show"
  },
  "ytlaVJ7heeU": {
    "album": null,
    "albumArtist": null,
    "artist": "Painface",
    "track": "On A Pale Horse"
  }
}
wrapper.js:74 storage.Options = {
  "disabledConnectors": "[hidden]",
  "forceRecognize": true,
  "scrobblePercent": 50,
  "scrobblePodcasts": true,
  "useNotifications": true,
  "useUnrecognizedSongNotifications": false
}
wrapper.js:74 storage.Connectors = {
  "YouTube": {
    "scrobbleEntertainmentOnly": false,
    "scrobbleMusicOnly": true
  }
}
wrapper.js:74 storage.ConnectorsOverrideOptions = {}
wrapper.js:74 storage.Core = {
  "appVersion": "2.73.0"
}
scrobble-service.js:69 Bind Last.fm scrobbler
scrobble-service.js:55 Unable to bind Libre.fm
bindAllScrobblers @ scrobble-service.js:55
scrobble-service.js:55 Unable to bind ListenBrainz
bindAllScrobblers @ scrobble-service.js:55
scrobble-service.js:55 Unable to bind Maloja
bindAllScrobblers @ scrobble-service.js:55
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube-music.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2078233019: Created controller for YouTube Music connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2078233021: Created controller for YouTube connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2078233020: Created controller for YouTube connector
util.js:40 Tab 2078233019: Remove controller for YouTube Music connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube-music.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2078233252: Created controller for YouTube Music connector
DevTools failed to load source map: Could not load content for chrome-extension://hhinaapppaileiechjoiifaancjggfjm/vendor/browser-polyfill.min.js.map: System error: net::ERR_FILE_NOT_FOUND
DevTools failed to load source map: Could not load content for chrome-extension://hhinaapppaileiechjoiifaancjggfjm/vendor/md5.min.js.map: System error: net::ERR_FILE_NOT_FOUND
```
**Environment (please complete the following information)**
-   OS: Windows
-   Browser: Chrome
-   Extension version: v2.73.0

**Additional context**
Stopped working after putting my computer to sleep, even restarting Chrome doesn't fix the issue

**Comment by jannielsen135** [comment]
I have the exact same issue

**Comment by duanyutong** [comment]
Same issue, unrelated to any sleep. The website may have had an update.

**Comment by JoshuaWalsh** [comment]
Strangely enough this happened to me mid-album. I didn't refresh the page or anything.

**Comment by richardDobron** [comment]
fixed, #3393
